### PR TITLE
Added list-unconsumed-queues action

### PR DESCRIPTION
**Pull request by afreiberger**
list-unconsumed-queues is an action performed by operations when receiving nrpe alerts for unconsumed queues.  This will be the basis of operating future actions purge-queue and delete-queue used for clearing out unconsumed queues such as when cinder-volume fanout queues rotate from service restarts and continue receiving messages bound for dead units.

### CODE REVIEW

**Comment by afreiberger** [comment]
If anyone has suggestions for tests, I'd welcome help.
I saw that the amulet creates a cinder relation, so I could imagine recycling the cinder-volume service to create a positive test result of a dead fanout queue, but couldn't determine how I'd actually exercise the systemctl restart cinder-volume on the cinder unit from within the test framework.

File: actions/actions.py [hunk 1]
```diff
@@ -41,9 +49,54 @@ def resume(args):
...
+    except Exception as ex:
+        # if no queues, just raises an exception
+        log(str(ex), level='DEBUG')
+        return []
+
+
+def list_unconsumed_queues(args):
+    """Resume the Ceilometer services.
+    @raises Exception should the service fail to start."""
+    count = 0
```
Comment (jillr) [inline actions/actions.py]: I think this is bad copypasta? ^^

File: actions/actions.py [hunk 2]
```diff
@@ -41,9 +49,54 @@ def resume(args):
...
+        for queue in vhost_queue_info(vhost):
+            if queue['consumers'] == 0:
+                vhostqueue = "unconsumed-queues.{}.{}".format(vhost, iterator)
+                action_set({vhostqueue: "{} - {}".format(queue['name'],
+                                                         queue['messages'])})
+                iterator = iterator + 1
+                count = count + 1
+
+    action_set({'unconsumed-queue-count': count})
+
```
Comment (jillr) [inline actions/actions.py:95]: You might consider action_fail'ing on exceptions.  Like: https://github.com/openstack/charm-percona-cluster/blob/master/actions/actions.py#L73

**Comment by jillr** [comment]
I think the subprocess.check_output would be a good case to mock and the rest could just be run in test.

File: actions/actions.py (hunk 1 above)
Comment (afreiberger) [reply actions/actions.py]: updated

**Pull request merged**